nameko_grpc/connection.py

```diff
@@ -52,18 +74,22 @@ class ConnectionManager:
     def __init__(self, sock, client_side):
         self.sock = sock
 
-        config = H2Configuration(client_side=client_side)
+        h2_logger = H2Logger(log.getChild("h2"))
+        config = H2Configuration(
+            client_side=client_side, logger=h2_logger
+        )  # logger type is wrong
```

Comment:
What does this comment mean?

Reply:
Ah yes terrible comment. H2Configuration types (via docstring) the logger class wrong. Will remove comment as it's not useful.